web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -50,102 +59,180 @@ const useStyles = makeStyles({
+                <TableCell>ISI</TableCell>
+                <TableCell>BUI</TableCell>
+                <TableCell>FWI</TableCell>
+                <TableCell>
+                  DGR
+                  <br />
+                  CL
+                </TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {Object.entries(props.fireCentres).map(([centreName, centre]) => {
+                return (
+                  <React.Fragment key={`fire-centre-${centreName}`}>
+                    <TableRow key={`fire-centre-${centreName}`}>
+                      <TableCell className={classes.fireCentre} colSpan={26}>
+                        {centre.name}
+                      </TableCell>
+                    </TableRow>
+                    {Object.entries(centre.planning_areas)
+                      .sort((a, b) => (a[1].name < b[1].name ? -1 : 1))
+                      .map(([areaName, area]) => {
+                        return (
+                          <React.Fragment key={`zone-${areaName}`}>
+                            <TableRow
+                              className={classes.planningArea}
+                              key={`zone-${areaName}`}
+                            >
+                              <TableCell className={classes.planningArea} colSpan={26}>
+                                {area.name}
+                              </TableCell>
+                            </TableRow>
+                            {Object.entries(area.stations)
+                              .sort((a, b) => (a[1].code < b[1].code ? -1 : 1))
+                              .map(([stationCode, station]) => {
+                                const daily = props.dailiesMap.get(station.code)
+                                return (
+                                  <TableRow
+                                    className={classes.station}
+                                    key={`station-${stationCode}`}
+                                  >
+                                    <TableCell key={`station-${station.code}-name`}>
+                                      {station.station_props.name} ({station.code})
+                                    </TableCell>
+                                    <TableCell key={`station-${station.code}-elevation`}>
+                                      {station.station_props.elevation}
+                                    </TableCell>
+                                    <TableCell key={`station-${station.code}-fuel-type`}>
+                                      {station.station_props.fuel_type.abbrev}
+                                    </TableCell>
+                                    <TableCell>{daily?.status}</TableCell>
+                                    <TableCell>{daily?.temperature}</TableCell>
+                                    <TableCell>{daily?.relative_humidity}</TableCell>
+                                    <TableCell>{daily?.wind_direction}</TableCell>
```

Comment:
I'm busy fixing wind direction to be three digits, no decimal places all over - can you please add it here for consistency.
```suggestion
                                    <TableCell>{daily?.wind_direction?.toFixed(0).padStart(3, '0')}</TableCell>
```

Reply:
Sure